packages/lesswrong/components/recentDiscussion/RecentDiscussionSubscribeReminder.tsx

```diff
@@ -160,57 +176,63 @@ const RecentDiscussionSubscribeReminder = ({classes}: {
       </AnalyticsInViewTracker>
     </AnalyticsContext>
   }
-  
+
   if (loading) {
-    return <div>
+    return <div className={classes.root}>
       <Loading/>
     </div>
   } else if (subscriptionConfirmed) {
     return <AnalyticsWrapper branch="already-subscribed">
-      You are subscribed to the best posts of LessWrong!
+      <div className={classes.message}>
+        <CheckRounded className={classes.checkIcon} />
+        You are subscribed to the best posts of {forumType}!
+      </div>
     </AnalyticsWrapper>
   } else if (!currentUser || adminBranch===0) {
     // Not logged in. Show a create-account form and a brief pitch.
     return <AnalyticsWrapper branch="logged-out">
       <div className={classes.message}>
+        <MailOutline className={classes.mailIcon} />
         To get the best posts emailed to you, create an account! {subscriptionDescription}
       </div>
       <div className={classes.loginForm}>
         <WrappedLoginForm startingState="signup" />
       </div>
       {adminUiMessage}
-      <div className={classes.buttons}>
-        {maybeLaterButton}
-        {dontAskAgainButton}
-      </div>
     </AnalyticsWrapper>
   } else if (!userHasEmailAddress(currentUser) || adminBranch===1) {
```

Comment:
This whole branch had issues (ex. the email address input lost focus after every keystroke). Now it has less issues but it's still not great. I saw we had around 20-30 users without email addresses, so I figured it's not really worth spending more time on this now.

Reply:
Yikes